### ISSUE TITLE
Multiple root folders

### ISSUE THREAD

**Issue by FixRM** [microsoft/infersharp#205]
Hello! Thank you for the great tool. Is there built it way to scan several "root" folders for the same solution. We have solution that consists of several libraries that designed to be executed as "plugins" by external host app, so there is no single composition root for that project. Another use case - are unit test projects. Of course, we are not going to ship their code but it is good to know if we have issues there.

I guess I can use the following `https://github.com/microsoft/infersharp/blob/main/RUNNING_INFERSHARP_ON_WINDOWS.md` to achieve my goal, but this requires some tricky path translations in case if I want it to run on any user machine.

**Comment by xinshiMSFT** [comment]
Hi, I think the quickest mitigation is to modify the execution script yourself. Basically the script copies all the .dll and .pdb files from an input path into a staging folder like [this](https://github.com/microsoft/infersharp/blob/2a604a4ed3e493e2ce5dfa1085bbe5aac78f7953/run_infersharp.sh#L41). You can modify it to allow multiple input paths and iterate through them with the same logic.

**Comment by FixRM** [comment]
I thought it sits inside WSL container and will be rewritten with next release?

**Comment by xinshiMSFT** [comment]
Yes, this is just a mitigation for you before our next release. You can modify the script in your WSL instance in the meantime. I will mark this as a feature request.

**Comment by FixRM** [comment]
Thank you
